aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLOperation.java

```diff
@@ -86,8 +86,8 @@ private AppSyncGraphQLOperation(
 
     @Override
     public void start() {
-        // No-op if start() is called post-execution
-        if (ongoingCall != null && ongoingCall.isExecuted()) {
+        // No-op if start() is called post-execution or canceled
+        if (ongoingCall != null && ((ongoingCall.isExecuted()) || ongoingCall.isCanceled())) {
```

Comment:
Is this added for precaution as the crash logs points that crash originates from onFailure.

Reply:
This line is not in the crash log, but isCanceled should also be checked here since the canceled call should not be started...